tutor/resources/styles/global/externals.less

```diff
@@ -1,4 +1,5 @@
 @import (less) '~react-datepicker/dist/react-datepicker.css';
 @import (less) '~fixed-data-table-2/dist/fixed-data-table.css';
+@import (css) '~react-joyride/lib/styles/react-joyride-compiled.css';
 
```

Comment:
This is causing errors since joyride isn't in package.json.  Was it added here by mistake?  I don't see it being used anywhere.